static/js/vue-cdr-access/src/mixins/gaUtils.js

```diff
@@ -0,0 +1,28 @@
+export default {
+    methods: {
+        pageEvent(recordData) {
+            let collection = recordData.briefObject.parentCollectionName || '';
+
+            if (collection === '' && recordData.briefObject.type === 'Collection') {
+                collection = recordData.briefObject.title;
+            }
+            if (collection === '') {
+                collection = '(no collection)';
+            }
+
+            this.$gtag.event('record', {
+                'event_category': recordData.briefObject.parentCollectionId,
+                'event_label': `${recordData.briefObject.title}|${recordData.briefObject.id}`,
+                'value': `${recordData.briefObject.title}|${recordData.briefObject.id}`
```

Comment:
Is the `value` needed here? Its being assigned the same as the even_label, and my understanding is that the `value` is intended for numerical values https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide

Reply:
Also, shouldn't the event have an action of "record"?

Reply:
The first parameter is the action, unless I'm reading this wrong, https://matteo-gabriele.gitbook.io/vue-gtag/methods/events

Reply:
ah, i missed the first parameter, ignore that